db/migrate/20150908084406_add_relationships_to_requests.rb

```diff
@@ -0,0 +1,21 @@
+class AddRelationshipsToRequests < ActiveRecord::Migration
+  def change
+    add_reference :requests, :user, references: :users, index: true
+    add_foreign_key :requests, :users
```

Comment:
You can replace these 2 lines with one line `add_reference :requests, :user, references: :users, index: true, foreign_key: true`